### PR TITLE
fix(arc-saas): pagination of components

### PR DESCRIPTION
**Pull request by Deepika516**
pagination of components

GH-34

## Description
pagination for 3 components named 
plan component,lead list componet, Tenant list component

Fixes #34 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: projects/saas-ui/src/app/on-boarding/components/add-tenant/add-tenant.component.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {BillingPlanService} from '../../../shared/services/billing-plan-service'
 export class AddTenantComponent implements OnInit {
   [x: string]: any;
   addTenantForm: FormGroup;
-  subscriptionPlans = [];
+  subscriptionPlans: any = [];
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/on-boarding/components/add-tenant/add-tenant.component.ts:19]: dont use any

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/saas-ui/src/app/on-boarding/components/add-tenant/add-tenant.component.ts (hunk 1 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/on-boarding/components/add-tenant/add-tenant.component.ts]: done

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts [hunk 2]
```diff
@@ -1,41 +1,59 @@
...
+  limit = 5;
   defaultColDef: ColDef = {
     flex: 1,
     minWidth: 150,
     filter: true,
     floatingFilter: true,
     resizable: true,
   };
-
+  keyVariable: any;
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts:38]: any 
please check and change its all occurance

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts (hunk 2 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/onboarding-tenant-list/onboarding-tenant-list.component.ts]: done

File: projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts [hunk 3]
```diff
@@ -53,18 +57,48 @@ export class ManagePlansComponent
...
+    params.api.setDatasource(dataSource);
   }
 
-  getPlans() {
-    this.billingPlanService
-      .getPlanOptions(null, null, this.filter)
-      .pipe(takeUntil(this._destroy$))
-      .subscribe(res => {
-        this.rowData = res.map(item => {
+  getPaginatedPlans(page: number, limit: number): Observable<any[]> {
```
Comment (yeshamavani) [inline projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts:93]: again same 
please check all files and remove

**Review by yeshamavani — CHANGES_REQUESTED** [review]

File: projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts (hunk 3 above)
Comment (Deepika516) [reply projects/saas-ui/src/app/main/components/manage-plans/manage-plans.component.ts]: sorry removed

**Review by yeshamavani — APPROVED** [review]

**Pull request merged**